.gitmodules

```diff
@@ -73,3 +73,9 @@
 [submodule "pack/programming/start/xml.vim"]
 	path = pack/programming/start/xml.vim
 	url = https://github.com/othree/xml.vim
+[submodule "pack/typescript/start/typescript-vim"]
+	path = pack/typescript/start/typescript-vim
```

Comment:
Can you please add the typescript plugin into `opt` instead of `start`. I don't usually use typescript and therefore don't want it to be loaded. It's already kinda bloated :)